04/app.js

```diff
@@ -0,0 +1,35 @@
+document.addEventListener('DOMContentLoaded', function () {
+  const divs = document.querySelectorAll('div');
+
+  const newClass = function (e) {
+    e.stopPropagation();
+    this.classList.add('clicked');
```

Comment:
Ta klasa ma zostać dodana dopiero po jakimś odstępie czasu więc jest to niepotrzebne.